Feature branch

direakanbi opened the pull request:
# My portfolio
This project was created for the creation of my portfolio.

# LANGUAGES USED
1. HTML
2. CSS
3. Linters(Lighthouse, Webhint, Stylelint
# PREREQUISITES
  1.Prior knowledge of HTML and CSS.

direakanbi committed:
My portfolio project.

direakanbi committed:
Update README.md

direakanbi committed:
Update styles.css

Alexoid1 commented on index.html:
```diff
@@ -0,0 +1,47 @@
+  <main>
+    <header>
+      <div class="header">
+       <h1> <a href="">My Logo</a></h1>
```
- [ ] Kindly remove that `h1` tag. The `h1` adds SEO support for your website so is important just to use it in the main title.

Alexoid1 replied:
- Add a # as the value in the `href` property. The active state in `a` tag changes the default color of the element in order to change this is common to use a # as value.

Alexoid1 commented on styles.css:
```diff
@@ -0,0 +1,74 @@
+  height: 20px;
+  text-align: center;
+  justify-content: space-between;
+  padding: 10%;
```
- [ ] Kindly use ṕadding values given in Figma template. Try using pixel values instead of percentage.
![Captura de pantalla_2021-09-17_02-26-44](https://user-images.githubusercontent.com/53111464/133742296-2cb4b06b-b223-4497-a063-b98d624a24e8.png)


**Figma**
![Captura de pantalla_2021-09-17_02-24-49](https://user-images.githubusercontent.com/53111464/133742132-3befa334-0b2a-45b4-a1c5-cc588fb96d9f.png)

Alexoid1 replied:
- [ ] Kindly remove that underline in the logo.

Alexoid1 commented on index.html:
```diff
@@ -0,0 +1,47 @@
+    </header>
+    <section class="container">
+      <div class="intro">
+        <header><h2>Hi! I'M DIRE</h2> 
```
- [ ] Delete the header tag.
- [ ] Replace `h2` with a` h1`. This is important to add SEO support to your website.

Alexoid1 commented on index.html:
```diff
@@ -0,0 +1,47 @@
+
+    
+       <div class="contact"> 
+          <h4>LET'S CONNECT</h4>
```
- [ ] Replace the `h4 `tag with a button or a link. 
`The button (if the template has one) is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)`.

- [ ] Align the LET'S CONNECT button with the paragraph at the top.

![Captura de pantalla_2021-09-17_03-03-43](https://user-images.githubusercontent.com/53111464/133747353-3b86bd9b-36cf-4e99-9fc6-afb3efe985cf.png).

- [ ] Kindly use the color values given in the Figma template.

Alexoid1 commented on README.md:
```diff
@@ -1,2 +1,9 @@
-This repo was made for the creation of my portfolio.
+# My portfolio
+This project was created for the creation of my portfolio.
+
```
- [ ] Add a link to a live version of the project.

Alexoid1 reviewed (changes requested):
#Hi @direakanbi ,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:
 - [ ] Add a link to a live version of the project.

For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

direakanbi committed:
Made changes to the HTML, CSS and README.

Modified the HTML to increase SEO.
Added the colors in relation to the template.
Added live link to the README.

direakanbi committed:
Update styles.css

SajjadAhmad14 commented on index.html:
```diff
@@ -0,0 +1,45 @@
+      <div class="intro">
+        <header><h1>Hi! I'M DIRE</h1> 
+          <h1>GLAD TO SEE <br>YOU!</h1>
+        </header> 
```
Please remove this header tag from here as suggested by the previous reviewer. Header tag should be used as the container for the Toolbar section.

SajjadAhmad14 commented on index.html:
```diff
@@ -0,0 +1,45 @@
+
+    
+       <div class="contact"> 
+          <a href="#">LET'S CONNECT</a>
```
Please use the same color as in the design for this link.

SajjadAhmad14 commented on index.html:
```diff
@@ -0,0 +1,45 @@
+       <a href="">My Logo</a>
+      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
+      </div> 
+    <section class="container">
```
[ OPTIONAL ] Please use all the images as the background which are provided on the design page.

SajjadAhmad14 reviewed (changes requested):
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Dire Akanbi, 
 ### Thank you for implementing changes suggested previously :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

direakanbi committed:
.

Mihndim2020 commented on index.html:
```diff
@@ -0,0 +1,44 @@
+</head>
+<body>
+  <main>
+      <div class="header">
```
- [OPTIONAL] This should be a `<header>` tag not a div with class header. Please kindly fix that.

Mihndim2020 reviewed (approved):
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @direakanbi , 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

![Congratulations-Memes-FEATURED](https://user-images.githubusercontent.com/65852621/133696300-b506e476-dc1a-443e-935a-c6f434e788b6.jpg)

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
